apps/tasks/pubspec.yaml

```diff
@@ -15,7 +15,7 @@ publish_to: 'none'
 # https://developer.apple.com/library/archive/documentation/General/Reference/InfoPlistKeyReference/Articles/CoreFoundationKeys.html
 # In Windows, build-name is used as the major, minor, and patch parts
 # of the product and file versions while build-number is used as the build suffix.
-version: 1.0.0+1
+version: 1.0.0+2
```

Comment:
It is fine, I guess. This gets override by flutter build?

Reply:
This is version increment is required as we already published two versions on the play store.
Already mentioned that, we need to get a proper versioning for all of our projects working asap.